### ISSUE TITLE
Lost gold selling linen cloth at AH / Realm: redemption

### ISSUE THREAD

**Issue by wensis** [WoWManiaUK/Redemption#2858]

**Comment by wensis** [comment]
Dear Aymelek and other GM's,

5 or more days ago i sold linen cloth at the AH. 
Aymelek asked me to report this issue trough this way. 

The solution Aymelek have provided have not been resulting in a positive outcome. 
I deleted the cache and waited a day. To bad nothing appeared in my mail box. 

This issue happened with more then 10 stacks of Linen cloth sold. 
I did recieve a comfirmation about selling linen cloth. But the mail had no attachment in the mail. 
Even after a couple of days nothing was recieved.

I use the latest update (with launcher) and as seen above deleted my cache. 

Is there anyway you guys could see my mail history and see why it wasn't send later on? 

Also this happened on 2 characters: Jefta and Wnss
Realm: Redemption

Maybe i'm the only one, but i cannot see why this would happen. 

Hope to hear from you guys soon! Thank you in advance!
